airflow/executors/kubernetes_executor.py

```
@@ -621,6 +223,8 @@ def execute_async(
             self.log.info("Add task %s with command %s", key, command)
 
         try:
+            from airflow.kubernetes.pod_generator import PodGenerator
```

Can we do this outside the try? If this failed (unlikely) it'd hide the real failure.

nit: It makes the most sense to me to put these right at the start of the method.

---

Sure, I tried to generally go with the philosophy of moving the imports as close to the usage as possible. Though, I can see/understand folks having lots of different personal preferences and pros/cons. Moving to the top of the method in this case is fine with me as long as we settle with that.

---

Yeah, like single vs double quotes. As long as it's out of the try, I'm good.